### PR TITLE
fix: update angular to 14

### PR DESCRIPTION
**Pull request by taro04**
@KimuraYu45z cc: @yutaro-elk 

This repository is used by `web-app`.
Therefore, I think this repository also needs to update Angular to 14.

I updated to 14 then, confirmed that the command `npm i` is executed correctly
 
prease review. 🙏

### CODE REVIEW

**Review by kimurayu45z — APPROVED** [review]
code is LGTM but why CI fail?

**Comment by ethan-massie-vectorsolutions** [comment]
> code is LGTM but why CI fail?

Looks like the CI needs to be updated to use node 14.15 or greater:

`The Angular CLI requires a minimum Node.js version of either v14.15, or v16.10.`

**Comment by taro04** [comment]
@KimuraYu45z 

Fixed build version of git workflow to 14.
There was a CI error that the `prod ` option did not exist.
I fixed it.

File: .github/workflows/nodejs.yml [hunk 1]
```diff
@@ -23,7 +23,7 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-node-
       - run: npm install
-      - run: npm run build -- --prod
+      - run: npm run build
```
Comment (taro04) [inline .github/workflows/nodejs.yml:26]: I don't know how this option came about. The `prod` option was not defined in package.json, so I deleted it.

**Pull request merged**